src/plugins/run/__tests__/__fixtures__/images/basic-inline.md

```diff
@@ -0,0 +1,18 @@
+Image in a paragraph: ![should have inline][ecologi_image].
```

Comment:
This markdown fixture captures the meat of the change in this PR. Basically a plugin to add "inline=true" to images that we shouldn't treat as block level elements in downstream renderers (like markdown-viewer).